Dockerfile

```
@@ -60,7 +60,7 @@ FROM base as final
 COPY --from=intermediate-builder /coursier_cache/ /coursier_cache/
 COPY --from=intermediate-builder /usr/local/share/jupyter/kernels/scala/ /usr/local/share/jupyter/kernels/scala/
 
-RUN chown -R bootcamp:bootcamp /chisel-bootcamp
+RUN chown -R bootcamp:bootcamp /chisel-bootcamp /coursier_cache
```

Wouldn't it make sense to use the ENV-var COURSIER_CACHE?
```suggestion
RUN chown -R bootcamp:bootcamp /chisel-bootcamp ${COURSIER_CACHE}
```